src/compiler/optimizations/utils.cc

```diff
@@ -38,7 +38,15 @@ bool is_This(Node* node, Class* holder, Method* method) {
   return target->as_Parameter()->index() == 0;
 }
 
-Type compute_guaranteed_type(Expression* node, Class* holder, Method* method) {
+static Type find_literal_type(List<Type> literal_types, Symbol symbol) {
+  for (auto literal_type : literal_types) {
+    if (literal_type.klass()->name() == symbol) return literal_type;
+  }
+  FATAL("Couldn't find literal type");
+  return Type::invalid();
```

Comment:
Is this return necessary? FATAL is fatal.

Reply:
I copied it from a different place, so I think yes. Most likely to silence the compiler.